charts/ingress-nginx/values.yaml

```
@@ -85,6 +85,8 @@ controller:
   enableTopologyAwareRouting: false
   # -- This configuration disable Nginx Controller Leader Election
   disableLeaderElection: false
+  # -- Duration a leader election is valid before it's getting re-elected. If not set, the default value defined on controller is 30s.
+  electionTTL:
```

Please do not introduce any more `null` default values. Specifying an empty string is better from a perspective of discoverability which type to use.

```suggestion
  # -- Duration a leader election is valid before it's getting re-elected. If not set, the default value defined on controller is 30s.
  electionTTL: ""
```